src/main/java/com/iridium/iridiumteams/configs/Messages.java

```diff
@@ -280,7 +281,7 @@ public Messages(String team, String command, String pluginName, String color) {
         createdWarp = "%prefix% &7Created warp %name%.";
         deletedWarp = "%prefix% &7%player% has deleted the warp %name%.";
         warpAlreadyExists = "%prefix% &7A warp with that name already exists.";
-        warpLimitReached = "%prefix% &7" + team + " Warp limit has been reached.";
+        warpLimitReached = "%prefix% &7" + team + " Warp limit has be   en reached.";
```

Comment:
```suggestion
        warpLimitReached = "%prefix% &7" + team + " Warp limit has been reached.";
```

Im assuming this was a typo lol